app/controllers/spree/admin/product_feeds_controller.rb

```diff
@@ -22,8 +22,8 @@ def new
       private
 
       def load_data
-        @product_catalogs = Spree::ProductCatalog.where(store: current_store).
-                            order(:name)
+        @product_catalogs = Spree::ProductCatalog.where(store: current_store).order(:name)
```

Comment:
it'd be cool to have a scope to fetch by store, something like `Spree::ProductCatalog.for_store(current_store)`

Reply:
done https://github.com/geminimvp/solidus_product_feed/pull/11/files#diff-154536b14bceed66fa7c2eec20e60704R25